src/app/filters/filter-fields.component.html

```diff
@@ -34,11 +35,38 @@
           <li *ngFor="let query of currentField.queries"
               [ngClass]="{'selected': query.value === currentValue}">
             <a role="menuitem" tabindex="-1" (click)="selectValue(query)">
+              <span *ngIf="query?.iconClass" class="fa {{query?.iconClass}}"></span>
+              <img class="avatar" *ngIf="query?.imageUrl" [attr.src]="query?.imageUrl" />
```

Comment:
I directly integrated it to planner and developed using watch:library. HTML doesn't update though, so I updated the HTML in node_modules of planner directly.